Flow.Launcher.Infrastructure/PinyinAlphabet.cs

```diff
@@ -59,7 +59,7 @@ public string Translate(string content)
 
         private string GetFirstPinyinChar(string content)
         {
-            return string.Concat(content.Split(';').Select(x => x.First()));
+            return string.Concat(content.Split(' ').Select(x => x.First()));
```

Comment:
need to add remove empty entries otherwise it will throw because you are doing First():
return string.Concat(content.Split(' ', StringSplitOptions.RemoveEmptyEntries).Select(x => x.First()));

Reply:
Well it isn't an issue before because the translated query won't contain something like ;;, but since we now use space to split. I will take a look on the translated query when the original query contain both Chinese and letter and space.